### PR TITLE
[4.4] Improve translation of aria label text by extra language keys

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #44040 .

### Summary of Changes
Use different language Keys for Pagination Buttons. This enables better translation. 

### Testing Instructions
See #44040. For english sites, The Aria Labels in pagination are correct before and after patch. 
Text can then be translated correcty.
For Example German "Gehe zur nächsten Seite" statt 
### Expected result AFTER applying this Pull Request
"Zur Seite weiter wechseln" is not a correct german sentence.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
I am surprised at the way the current code creates the language strings. I can't see why it was created that way.
I am also surprised that no translation team has raised this issue before.

As the text in the core doesn't change and at least one language will be improved by this change then its all good by me

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ac0e2ebcc58434f68744e39467c882aae6f78e3e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44050">issues.joomla.org/tracker/joomla-cms/44050</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on ac0e2ebcc58434f68744e39467c882aae6f78e3e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44050">issues.joomla.org/tracker/joomla-cms/44050</a>.</sub>

File: layouts/joomla/pagination/link.php [hunk 1]
```diff
@@ -21,26 +21,25 @@
     // Check for "Start" item
     case Text::_('JLIB_HTML_START'):
         $icon = $app->getLanguage()->isRtl() ? 'icon-angle-double-right' : 'icon-angle-double-left';
-        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION', strtolower($item->text));
+        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION_START');
```
Comment (sandewt) [inline layouts/joomla/pagination/link.php]:
```suggestion
        $aria = Text::_('JLIB_HTML_GOTO_POSITION_START');
```

File: layouts/joomla/pagination/link.php [hunk 2]
```diff
@@ -21,26 +21,25 @@
...
         break;
 
     // Check for "Prev" item
     case $item->text === Text::_('JPREV'):
-        $item->text = Text::_('JPREVIOUS');
         $icon = $app->getLanguage()->isRtl() ? 'icon-angle-right' : 'icon-angle-left';
-        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION', strtolower($item->text));
+        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION_PREVIOUS');
```
Comment (sandewt) [inline layouts/joomla/pagination/link.php]:
```suggestion
        $aria = Text::_('JLIB_HTML_GOTO_POSITION_PREVIOUS');
```

File: layouts/joomla/pagination/link.php [hunk 3]
```diff
@@ -21,26 +21,25 @@
...
         break;
 
     // Check for "Next" item
     case Text::_('JNEXT'):
         $icon = $app->getLanguage()->isRtl() ? 'icon-angle-left' : 'icon-angle-right';
-        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION', strtolower($item->text));
+        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION_NEXT');
```
Comment (sandewt) [inline layouts/joomla/pagination/link.php]:
```suggestion
        $aria = Text::_('JLIB_HTML_GOTO_POSITION_NEXT');
```

File: layouts/joomla/pagination/link.php [hunk 4]
```diff
@@ -21,26 +21,25 @@
...
         break;
 
     // Check for "End" item
     case Text::_('JLIB_HTML_END'):
         $icon = $app->getLanguage()->isRtl() ? 'icon-angle-double-left' : 'icon-angle-double-right';
-        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION', strtolower($item->text));
+        $aria = Text::sprintf('JLIB_HTML_GOTO_POSITION_END');
```
Comment (sandewt) [inline layouts/joomla/pagination/link.php]:
```suggestion
        $aria = Text::_('JLIB_HTML_GOTO_POSITION_END');
```

**Comment by brianteeman** [comment]
oops I missed the code change that @sandewt spotted.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9fa34639112961ec798dc8f7e8ba5b9d13ab8ab0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44050">issues.joomla.org/tracker/joomla-cms/44050</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44050">issues.joomla.org/tracker/joomla-cms/44050</a>.</sub>

**Review by tecpromotion — APPROVED** [review]
Thanks @chmst

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**